lib/fluent/plugin/prometheus_metrics.rb

```diff
@@ -0,0 +1,79 @@
+module Fluent::Plugin
+  class PromMetricsAggregator
+    def initialize
+      @known_metrics = []
+      @merged_lines = []
+    end
+
+    def get_metric_name(line)
+      metric = ''
+      tokens = line.split(' ')
+      if line[0] == '#'
+        if ['HELP', 'TYPE'].include?(tokens[1])
+          metric = tokens[2]
+        end
+      else
+        metric = tokens[0].split('{')[0]
+      end
+      metric
+    end
+
+    def is_meta_line(line)
+      tokens = line.split(' ')
+      line[0] == '#' && ['HELP', 'TYPE'].include?(tokens[1])
+    end
+
+    def add_metrics(metrics)
```

Comment:
Put this link somewhere. 
https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md

Reply:
Added in 3eff52fd5dd63c1cb429a2a666fa7bdb2fde48e2